### PR TITLE
Sampling from variational posterior

### PR DESCRIPTION
**Pull request by taku-y**
Add a function to draw samples from variational posterior.

### CODE REVIEW

File: pymc3/variational/advi.py [hunk 1]
```diff
@@ -294,3 +296,53 @@ def adagrad(grad, param, learning_rate, epsilon, n):
     updates[param] = param - (-learning_rate * grad /
                               theano.tensor.sqrt(accu_sum + epsilon))
     return updates
+
+def sample_vp(draws, model, vparams, seed=1):
+    """Draw samples from variational posterior.
```
Comment (fonnesbeck) [inline pymc3/variational/advi.py:301]: the pymc3 conventional argument is `random_seed` rather than `seed`

File: pymc3/variational/advi.py [hunk 2]
```diff
@@ -294,3 +296,53 @@ def adagrad(grad, param, learning_rate, epsilon, n):
...
+        Estimated variational parameters of the model. 
+    seed : int
+        Seed of random number generator. 
+
+    Returns
+    -------
+    trace : pymc3.backends.base.MultiTrace
+        Samples drawn from the variational posterior. 
+    """
+    if type(vparams) is ADVIFit:
```
Comment (fonnesbeck) [inline pymc3/variational/advi.py]: use `isinstance` instead of `type ... is`

File: pymc3/variational/advi.py [hunk 3]
```diff
@@ -294,3 +296,53 @@ def adagrad(grad, param, learning_rate, epsilon, n):
     updates[param] = param - (-learning_rate * grad /
                               theano.tensor.sqrt(accu_sum + epsilon))
     return updates
+
+def sample_vp(draws, model, vparams, random_seed=20090425):
```
Comment (twiecki) [inline pymc3/variational/advi.py]: I think you can use `model=None` and then try to get it from the `modelcontext` to enable the `with` context manager.

**Comment by twiecki** [comment]
Thanks @taku-y, this is great!
